### PR TITLE
PG3BUG/space-on-accounts

### PR DESCRIPTION
**Pull request by lsanchezd**
* Change a few styles for intersys font on account cards
* Add new styles for the border account cards 
* Solving the space between the cards.

### CODE REVIEW

**Comment by mmartinezintersys** [comment]
https://trello.com/c/YtbGSq7o/216-p3-bug-accounts-cards-are-not-arragend-properly-when-having-long-account-names

File: account/templates/account.html [hunk 1]
```diff
@@ -368,7 +369,8 @@ <h5 class="modal-title" id="modalProjectLabel">Create new account</h5>
             
               <div class="form-group">
                 <label for="recipient-name" class="col-form-label">Name of the new account:</label>
-                <input  type="text" class="form-control" id="new_account_name" name="name" maxlength=40 pattern="\S+" required>
+                <input  type="text" class="form-control" id="new_account_name" name="name" maxlength=40 required>
+                <!--pattern="\S+"-->
```
Comment (mmartinezintersys) [inline account/templates/account.html]: There is a fix for this pattern on develop, so be sure to include it during the merge.

File: account/templates/account.html [hunk 2]
```diff
@@ -399,7 +401,8 @@ <h5 class="modal-title" id="modalProjectLabel">Create new project</h5>
               </div>
               <div class="form-group">
                 <label for="new_project_name" class="col-form-label">New project:</label>
-                <input type="text" class="form-control" id="new_project_name" name="name" maxlength=40 pattern="\S+" required>
+                <input type="text" class="form-control" id="new_project_name" name="name" maxlength=40  required>
+                <!--pattern="\S+"-->
```
Comment (mmartinezintersys) [inline account/templates/account.html]: Same as line 373

File: account/templates/account.html (hunk 1 above)
Comment (lsanchezd) [reply account/templates/account.html]: I leave the develop pattern which was ok

**Pull request merged**